rootfs/etc/nginx/lua/test/configuration_test.lua

```diff
@@ -0,0 +1,66 @@
+package.path = "./rootfs/etc/nginx/lua/?.lua;./rootfs/etc/nginx/lua/test/mocks/?.lua;" .. package.path
```

Comment:
Not really feedback on this PR but - is there a reason we need to include this line in every lua test file? Same with `_G.TEST = true`? 

Could we use another test_helper module to normalize this?

Reply:
What's the default `package.path` before you modify it?